perf/perf_pcp.py

```diff
@@ -89,12 +89,16 @@ def test_perfevent_24x7_events(self):
         self.lpar_24x7_ppc64le_check()
         if self.cpu_family == 'power8':
             self.cancel("Not supported on Power8")
-        # -s 2 collects two samples
-        cmd = "pmval -s 2 perfevent.hwcounters.hv_24x7.PM_MBA0_CLK_CYC.value"
-        process.run(cmd, shell=True)
-        # -s 2 collects two samples
-        cmd = "pmval -s 2 perfevent.hwcounters.hv_24x7.PM_PB_CYC.value"
-        process.run(cmd, shell=True)
+        output = process.system_output("pminfo").decode()
+        for line in output.splitlines():
+            # -s 2 collects two samples
+            if 'PM_MBA0_CLK_CYC' in line and 'value' in line:
+                process.run('pmval -s 2 %s' % line, shell=True)
+                break
```

Comment:
Why is this added? It never reaches the below code in loop with this added.

Reply:
Removed break...